lib/builders/observability-builder.ts

```diff
@@ -0,0 +1,77 @@
+import * as blueprints from '../../lib';
+import * as utils from "../utils";
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class ObservabilityBuilder extends blueprints.BlueprintBuilder {
+
+    public addNativeObservabilityBuilderAddOns(): ObservabilityBuilder {
+        return this.addOns(
+            new blueprints.addons.AwsLoadBalancerControllerAddOn(),
+            new blueprints.addons.CertManagerAddOn(),
+            new blueprints.addons.CoreDnsAddOn(),
+            new blueprints.addons.KubeProxyAddOn(),
+            new blueprints.addons.KubeStateMetricsAddOn(),
+            new blueprints.addons.MetricsServerAddOn(),
+            new blueprints.addons.PrometheusNodeExporterAddOn(),
+            new blueprints.addons.VpcCniAddOn());
+    }
+
+    public addMixedObservabilityBuilderAddOns(): ObservabilityBuilder {
```

Comment:
method name is too verbose, we should remove `builder` from it as this class already has the word builder in it. I also propose dropping `observability` from the method names for the same reason. 
 
If you add the actual observability addons then I propose `addMixedPatternAddons`.

If there are only prereqs and not the actual working solution, then change `add` to `enable`.

Reply:
Fixed